app/channels/notifications_channel.rb

```diff
@@ -1,9 +1,6 @@
 class NotificationsChannel < ApplicationCable::Channel
-  def self.broadcast_changed(user, count: user.notifications.unread.count)
-    NotificationsChannel.broadcast_to(user, {
-                                        type: "notifications.changed",
-                                        payload: { count: count }
-                                      })
+  def self.broadcast_changed(user)
+    NotificationsChannel.broadcast_to(user, { type: "notifications.changed" })
```

Comment:
I've removed the count as it's safer for us to just refetch via the API when a websocket message comes in.